sources/engine/Stride.Engine/Engine/Gizmos/IEntityGizmo.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
New files shouldn't mention SiliconStudio, unless we consider it is the same class moved from `Stride.Assets.Presentation.AssetEditors.Gizmos` in which case we keep it.